runtime/compiler/il/J9Symbol.cpp

```diff
@@ -382,3 +451,10 @@ template TR::Symbol * J9::Symbol::createRecognizedShadow(TR_StackMemory,
 template TR::Symbol * J9::Symbol::createRecognizedShadow(TR_HeapMemory,          TR::DataType, uint32_t, RecognizedField);
 template TR::Symbol * J9::Symbol::createRecognizedShadow(PERSISTENT_NEW_DECLARE, TR::DataType, uint32_t, RecognizedField);
 
+template TR::Symbol * J9::Symbol::createPossiblyRecognizedShadowWithFlags(TR_StackMemory,         TR::DataType, bool, bool, bool, RecognizedField);
+template TR::Symbol * J9::Symbol::createPossiblyRecognizedShadowWithFlags(TR_HeapMemory,          TR::DataType, bool, bool, bool, RecognizedField);
+template TR::Symbol * J9::Symbol::createPossiblyRecognizedShadowWithFlags(PERSISTENT_NEW_DECLARE, TR::DataType, bool, bool, bool, RecognizedField);
+
+template TR::Symbol * J9::Symbol::createPossiblyRecognizedShadowFromCP(TR::Compilation*, TR_StackMemory,         TR_ResolvedMethod*, int32_t, TR::DataType*, uint32_t*, bool);
+template TR::Symbol * J9::Symbol::createPossiblyRecognizedShadowFromCP(TR::Compilation*, TR_HeapMemory,          TR_ResolvedMethod*, int32_t, TR::DataType*, uint32_t*, bool);
+template TR::Symbol * J9::Symbol::createPossiblyRecognizedShadowFromCP(TR::Compilation*, PERSISTENT_NEW_DECLARE, TR_ResolvedMethod*, int32_t, TR::DataType*, uint32_t*, bool);
```

Comment:
Minor nit.  Is allocation in persistent memory something that is likely to happen?  It is technically possible to do, but is it actually useful?  If not, I'd say remove these specializations for persistent memory.

Reply:
I've amended the commit so that it no longer introduces the persistent memory specializations, and I've added another commit to remove the ones that were already there for `createRecognizedShadow()`.